test/unit/dialects/mysql/query.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
I would recommend integration test for this, you can use this https://github.com/sequelize/sequelize/issues/9919#issuecomment-420938123 or create your own. (Model.create part is important)

Create a new file called `warning.test.js` [here](https://github.com/sequelize/sequelize/tree/master/test/integration/dialects/mysql)

1. Move this test https://github.com/sequelize/sequelize/blob/master/test/integration/sequelize.test.js#L305
2. Add a new one for `Model.create` case